src/ess/amor/resolution.py

```diff
@@ -120,10 +120,11 @@ def angular_resolution(
 
```

Comment:
Looking at the implementation of the wavelength resolution, we have
```Py
distance_between_choppers = (
    chopper_2_position.fields.z - chopper_1_position.fields.z
)
chopper_detector_distance = pixel_position.fields.z - chopper_midpoint.fields.z
```
Maybe it should be changed to
```Py
distance_between_choppers = sc.norm(chopper_2_position - chopper_1_position)
chopper_detector_distance = sc.norm(pixel_position - chopper_midpoint)
```
?